packages/objection/spec/interpreters.spec.ts

```diff
@@ -0,0 +1,258 @@
+import { Model } from 'objection'
+import { FieldCondition as Field, CompoundCondition } from '@ucast/core'
+import Knex from 'knex'
+import { expect } from './specHelper'
+import { createObjectionInterpreter, $eq, $ne, $lte, $lt, $gte, $gt, $exists, $in, $nin, $and, $not, $or } from '../src'
+
+const knex = Knex({ client: 'pg' })
```

Comment:
probably `knex`, `objection` setup and `User`, `Project classes would be better to move into `spec/fixtures.js`. To reduce the cognitive complexity of tests.So, eventually

```js
import { Project, User } from ### './fixtures'
```

What are you thoughts on this?

Reply:
Sounds good.